DemoApp/ChatPresenter.swift

```diff
@@ -15,6 +15,8 @@ class DemoChatChannelListRouter: ChatChannelListRouter {
 
     var channelPresentingStyle: ChannelPresentingStyle = .push
 
+    let streamModalTransitioningDelegate = StreamModalTransitioningDelegate()
```

Comment:
nitpick:
This could be lazy var

Reply:
What is this benefit in this case? 🤔 The delegate is pretty cheap to create, the Presentation Controller is only created when the VC is presented

Reply:
It's created everytime, irregardless whether you're changing the presentation style. The default presentation style is `.push`, so it's kinda useless to have it in place.

Reply:
✅